src/main/java/uk/gov/beis/els/controller/StaticContentController.java

```diff
@@ -13,4 +13,11 @@ public ModelAndView renderAccessibilityStatement() {
     return new ModelAndView("staticContent/accessibilityStatement");
   }
 
+  @GetMapping("/cookies")
+  public ModelAndView renderCookiePolicy(@RequestHeader(value = "referer", required = false) final String referer) {
+    ModelAndView modelAndView = new ModelAndView("staticContent/cookiePolicy");
+    modelAndView.addObject("referer", referer);
```

Comment:
Only a minor issue but we should probably only present redirect links to referrers from our own domain as a security measure. We can add a config option for the allowed domain and if the referrer domain is different just link to the start page or something.